docs/.vitepress/config.js

```diff
@@ -31,7 +31,7 @@ export default defineConfig({
       'link',
       {
         rel: 'stylesheet',
-        href: 'https://assets.finn.no/pkg/@warp-ds/tokens/v1/finn-no.css',
+        href: 'https://assets.finn.no/pkg/@warp-ds/css/v1/tokens/finn-no.css',
```

Comment:
I think we also need to add [fonts](https://assets.finn.no/pkg/@warp-ds/fonts/v1/finn-no.css) here now that they are no longer included in the brand CSS. I suppose it would be apparent in the [Font Size page](https://warp-ds.github.io/css-docs/font-size#setting-the-font-size).